models/articles-models.js

```diff
@@ -1,18 +1,31 @@
 const db = require('../db/connection');
+const format = require('pg-format');
+
+const checkExists = async (table, column, value) => {
```

Comment:
I'd really like to see this written consistently - why have you used async/await here and .then further down?